docs/pages/components/index.md

```diff
@@ -0,0 +1,77 @@
+# NFT Components
```

Comment:
I really don't like the `NFT` prefix but not blocking:
```suggestion
# Components
```

Same comment on `NFT Templates`

Reply:
Component is a bit unclear (but agree that nft is not great) maybe UI components/UI templates

Reply:
done, all renamed with UI components/UI templates